src/DataViews/SamplingReportDataView.cpp

```diff
@@ -343,39 +342,73 @@ void SamplingReportDataView::SetStackEventsCount(uint32_t stack_events_count) {
   stack_events_count_ = stack_events_count;
 }
 
-std::string SamplingReportDataView::GetToolTip(int row, int column) {
-  if (column != kColumnInclusive && column != kColumnExclusive) {
-    return "";
-  }
-  const SampledFunction& function = GetSampledFunction(row);
-  uint32_t raw_count = {};
-  float percentage = {};
-  std::string count_type;
-  std::string at_the_top_or_encountered;
-  if (column == kColumnInclusive) {
-    raw_count = function.inclusive;
-    percentage = function.inclusive_percent;
-    count_type = "inclusive";
-    at_the_top_or_encountered = "encountered";
-  } else {
-    raw_count = function.exclusive;
-    percentage = function.exclusive_percent;
-    count_type = "exclusive";
-    at_the_top_or_encountered = "at the top of the callstack";
-  }
-
-  orbit_statistics::BinomialConfidenceInterval interval =
-      app_->GetConfidenceIntervalEstimator().Estimate(percentage / 100.0f, stack_events_count_);
-
+[[nodiscard]] static std::string BuildTooltipTail(
```

Comment:
Maybe the `[[nodiscard]]` is unnecessary here in the cpp file. And the same for the follwing BuildToolTip* methods.

Reply:
But I like `nodiscard` on pure functions.